🪲Acrobat Reader CPEs don't match expectations

**Fleet version**: NVD feed 2025-05-29

<hr/>

### 💥  Actual behavior

```
=== RUN   TestCPEFromSoftwareIntegration
    cpe_test.go:1816: Using /Users/ian/Downloads/cpe-5806690027c4868a4abef49f92654589907904436b2633e81b1b78ab118627d7.sqlite as database file
    cpe_test.go:1832: 
        	Error Trace:	/Volumes/Code/CH/fleet/server/vulnerabilities/nvd/cpe_test.go:1832
        	Error:      	Not equal: 
        	            	expected: "cpe:2.3:a:adobe:acrobat_reader_dc:22.002.20191:*:*:*:*:macos:*:*"
        	            	actual  : "cpe:2.3:a:adobe:acrobat_reader:22.002.20191:*:*:*:*:macos:*:*"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-cpe:2.3:a:adobe:acrobat_reader_dc:22.002.20191:*:*:*:*:macos:*:*
        	            	+cpe:2.3:a:adobe:acrobat_reader:22.002.20191:*:*:*:*:macos:*:*
        	Test:       	TestCPEFromSoftwareIntegration
        	Messages:   	Adobe Acrobat Reader DC.app
    cpe_test.go:1832: 
        	Error Trace:	/Volumes/Code/CH/fleet/server/vulnerabilities/nvd/cpe_test.go:1832
        	Error:      	Not equal: 
        	            	expected: "cpe:2.3:a:adobe:acrobat_dc:22.002.20212:*:*:*:*:windows:*:*"
        	            	actual  : ""
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-cpe:2.3:a:adobe:acrobat_dc:22.002.20212:*:*:*:*:windows:*:*
        	            	+
        	Test:       	TestCPEFromSoftwareIntegration
        	Messages:   	Adobe Acrobat DC (64-bit)
--- FAIL: TestCPEFromSoftwareIntegration (3.16s)
```

### 🧑‍💻  Steps to reproduce

Run `TestCPEFromSoftwareIntegration` with Adobe Acrobat DC entries uncommented.

### 🕯️ More info

Something changed in the May 29 build of the CPE database to break these CPEs. May 28 CPE DB behaves as expected.

### 🛠️ To fix
Research needs to be done to determine the fix.

--- a/server/vulnerabilities/nvd/cpe_test.go
+++ b/server/vulnerabilities/nvd/cpe_test.go
@@ -492,18 +492,18 @@ func TestCPEFromSoftwareIntegration(t *testing.T) {
 	testCases := []struct {
 		software fleet.Software
 		cpe      string
-	}{
-		{
-			software: fleet.Software{
-				Name:             "Adobe Acrobat Reader DC.app",
-				Source:           "apps",
-				Version:          "22.002.20191",
-				Vendor:           "",
-				BundleIdentifier: "com.adobe.Reader",
-			},
-			cpe: "cpe:2.3:a:adobe:acrobat_reader_dc:22.002.20191:*:*:*:*:macos:*:*",
-		},
-		{
+	}{ /*
+			{ // See #29570
+					software: fleet.Software{
+						Name:             "Adobe Acrobat Reader DC.app",
+						Source:           "apps",
+						Version:          "22.002.20191",
+						Vendor:           "",
+						BundleIdentifier: "com.adobe.Reader",
+					},
+					cpe: "cpe:2.3:a:adobe:acrobat_reader_dc:22.002.20191:*:*:*:*:macos:*:*",
+				},
+		*/{
 			software: fleet.Software{
 				Name:             "Adobe Lightroom.app",
 				Source:           "apps",
@@ -782,8 +782,8 @@ func TestCPEFromSoftwareIntegration(t *testing.T) {
 				Vendor:           "Igor Pavlov",
 				BundleIdentifier: "",
 			}, cpe: "cpe:2.3:a:7-zip:7-zip:22.01:*:*:*:*:windows:*:*",
-		},
-		{
+		}, /*
+			{ // See #29570
 			software: fleet.Software{
 				Name:             "Adobe Acrobat DC (64-bit)",
 				Source:           "programs",
@@ -791,8 +791,8 @@ func TestCPEFromSoftwareIntegration(t *testing.T) {
 				Vendor:           "Adobe",
 				BundleIdentifier: "",
 			}, cpe: "cpe:2.3:a:adobe:acrobat_dc:22.002.20212:*:*:*:*:windows:*:*",
-		},
-		{
+			},
+		*/{
 			software: fleet.Software{
 				Name:             "Brave",
 				Source:           "programs",

